go.mod

```diff
@@ -6,5 +6,4 @@ require (
 	github.com/gopherjs/gopherjs v0.0.0-20210202160940-bed99a852dfe // indirect
```

Comment:
It is worth updating this repo to use go version 1.17

ci files will also need updating to 1.17.1 for build.yml, unit.yml and component.yml (if exists) and then audit.yml and lint.yml should use `latest` as the version.

Reply:
👍